crates/ruma-events/src/image_pack.rs

```diff
@@ -0,0 +1,103 @@
+//! Types for image packs in Matrix ([MSC2545]).
+//!
+//! [MSC2545]: https://github.com/matrix-org/matrix-spec-proposals/pull/2545
+
+use std::collections::{BTreeMap, BTreeSet};
+
+use ruma_common::{serde::StringEnum, OwnedMxcUri, OwnedRoomId};
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+use crate::{room::ImageInfo, PrivOwnedStr};
+
+/// The content of an `im.ponies.room_emotes` event.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
+#[ruma_event(type = "im.ponies.room_emotes", kind = State, state_key_type = String)]
+pub struct RoomImagePackEventContent {
+    /// A list of images avaliable.
```

Comment:
Can you add to the docs that the keys in the map are the shortcodes for the images?